File: 0242-valid-anagram/0242-valid-anagram.java
public class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()) return false;

        int[] count = new int[26];  // For lowercase letters a-z

        for (int i = 0; i < s.length(); i++) {
            count[s.charAt(i) - 'a']++;  // Increase count for s
            count[t.charAt(i) - 'a']--;  // Decrease count for t
        }

        for (int i : count) {
            if (i != 0) return false;
        }

        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        int s1=s.Length;
        int t1=t.Length;
        if(s1!=t1){
            return false;
        }
        char[] arr=new char[s1];
        for(int i=0;i<s1;i++){
            arr[i]=s[i];
            
        }
        char[] arr1=new char[t1];
        for(int i=0;i<t1;i++){
            arr1[i]=t[i];
            
        }
        Array.Sort(arr);  
        Array.Sort(arr1);
        bool b=true;
        for(int i=0;i<t1;i++){
            if(arr[i]!=arr1[i]){
                b=false;

            }
        }
        return b;
    }

}